com.blitterstudio.amiberry.yml

```diff
@@ -0,0 +1,116 @@
+  - --share=network
+  # Allow bluetooth (e.g. controllers)
+  - --allow=bluetooth
+  # Play sounds using PulseAudio
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  
+modules:
+  - name: libmpeg2
+    config-opts:
+      - --disable-static
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
+        sha256: dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: libserialport
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.a
+    sources:
+      - type: archive
+        url: https://github.com/BlitterStudio/amiberry/raw/master/external/libserialport/libserialport-0.1.1.tar.gz
+        sha256: 4e3573e3bdbb6b0a872f32c4826d1b6b630dcc3eeb5f05d0501bb79d738a09ea
+
+  - name: amiberry
+    only-arches:
+      - x86_64
+    post-install:
+      - install -Dp -m 755 amiberry /app/bin/amiberry
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    builddir: true
+    
+    sources:
+      - type: archive
+        url: https://github.com/BlitterStudio/amiberry/archive/refs/tags/v5.6.2.tar.gz
+        sha256: 4e9feb3ab0039871a25dda9f42a29c9ec1d6160ff68ef7c37fd3205f59c14b97
+
+  - name: desktop
+    buildsystem: simple
+    build-commands:
+      - install -D -m644 -t /app/share/applications/ ${FLATPAK_ID}.desktop
+      - install -D -m644 -t /app/share/metainfo/ ${FLATPAK_ID}.metainfo.xml
+      - install -D -m755 -t /app/bin/ amiberry.sh
+      - for i in 32 48 64 128 256;do install -Dm644 ${i}x${i}.png /app/share/icons/hicolor/${i}x${i}/apps/amiberry.png;done
+#     - install -Dm644 com.blitterstudio.amiberry.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - mkdir -p /app/data
+      - cp -R abr /app/data
+      - cp -R conf /app/data
+      - cp -R controllers /app/data
+      - cp -R data /app/data
+      - cp -R inputrecordings /app/data
+      - cp -R kickstarts /app/data
+      - cp -R nvram /app/data
+      - cp -R savestates /app/data
+      - cp -R screenshots /app/data
+      - cp -R whdboot /app/data
+    sources:
+      - type: file
+        path: com.blitterstudio.amiberry.desktop
+      - type: file
+        path: com.blitterstudio.amiberry.metainfo.xml
+      - type: dir
+        path: ..
+      - type: file
+        path: com.blitterstudio.amiberry.metainfo.xml
+      - type: file
+        path: com.blitterstudio.amiberry.desktop
+      - type: file
+        path: 32x32.png
+      - type: file
+        path: 48x48.png
+      - type: file
+        path: 64x64.png
+      - type: file
+        path: 128x128.png
+      - type: file
+        path: 256x256.png
+#     - type: file
+#       path: com.blitterstudio.amiberry.svg
```

Comment:
why not using a SVG?

Reply:
I don't have one to use, unfortunately. The project is mostly a one-man show.

Reply:
the comment led to think there was one.

Reply:
Sorry, I added it because I was hoping to get one from a member of the community, but so far it hasn't happened. I can remove it if you feel that's problematic

Reply:
it gave the wrong signal. Don't need it? Don't put it.